message-me/app/javascript/packs/application.js

```diff
@@ -14,4 +14,7 @@ Turbolinks.start()
 ActiveStorage.start()
 $(document).on('turbolinks:load', function() {
   $('.ui.dropdown').dropdown();
+  $('.message .close').on('click', function() {
```

Comment:
1. Why must click on the icon only can close?
Isn't it bad UX when you must move the mouse specifically to the icon to close it?

2. Why do you put this within `$(document).on('turbolinks:load'` ? Why not outside?

Reply:
1. This is from the Semantic-ui code
![image](https://user-images.githubusercontent.com/67599130/126860628-83d34e9b-0c50-4eea-9428-181b91c40daf.png)
This is the message in action. 
![image](https://user-images.githubusercontent.com/67599130/126860635-e3164d4d-d505-43cd-9ed7-666646036336.png)
So your suggestion is that it fades automatically after few seconds? 

2. Putting it outside will not be able to close the message after it appears, because Turbolinks overrides the normal page loading process, the event that this relies upon will not be fired.

Reply:
![gif](https://user-images.githubusercontent.com/67599130/126861287-1b76f12a-03de-4fb0-9356-1d9b0af046b5.gif)
Is this better?

Reply:
1. wut? So what if its semantic-ui code? What I am saying is:
![image](https://user-images.githubusercontent.com/5483026/126862869-adf55a74-e952-47bf-81cc-4b3f94bbf44d.png)
Your previous code says, click on this "X" then close the entire msg. Why not click anywhere on the message container to close the message?
![image](https://user-images.githubusercontent.com/5483026/126862879-80599e1e-b6b9-48d1-8ab5-2ce71aa1899d.png)

---
Your change basically means a complete change in functionality. So, its completely different from what I am talking about

2. Why the event won't fire? Your answer is not accurate

Reply:
2. We need to wait for the `DOM` to load before we run any functions, since rails uses `turbolink`, we have to use the `turbolinks:load` event, which is called the first time on page load and every time on a `turbolink visit`. Only after it loads, the function within it can execute. This is what I found.